### PR TITLE
removed en-US in order to fix https://bugzilla.mozilla.org/show_bug.cgi?id=1118737

### PR DESCRIPTION
**Pull request by pollti**

### CODE REVIEW

File: themes/OneMozilla/footer.php [hunk 1]
```diff
@@ -17,9 +17,9 @@
     <nav id="nav-meta">
       <ul role="navigation">
         <li><a href="https://www.mozilla.org/contact/" rel="external"><?php _e('Contact Us', 'onemozilla'); ?></a></li>
-        <li><a href="https://www.mozilla.org/en-US/privacy/" rel="external"><?php _e('Privacy Policy', 'onemozilla'); ?></a></li>
-        <li><a href="https://www.mozilla.org/en-US/about/legal.html" rel="external"><?php _e('Legal Notices', 'onemozilla'); ?></a></li>
-        <li><a href="https://www.mozilla.org/en-US/legal/fraud-report/index.html" rel="external"><?php _e('Report Trademark Abuse', 'onemozilla'); ?></a></li>
+        <li><a href="https://www.mozilla.org/privacy/" rel="external"><?php _e('Privacy Policy', 'onemozilla'); ?></a></li>
+        <li><a href="https://www.mozilla.org/about/legal.html" rel="external"><?php _e('Legal Notices', 'onemozilla'); ?></a></li>
```
Comment (craigcook) [inline themes/OneMozilla/footer.php]: This link can be changed to 'https://www.mozilla.org/about/legal/'

File: themes/OneMozilla/footer.php [hunk 2]
```diff
@@ -17,9 +17,9 @@
...
+        <li><a href="https://www.mozilla.org/legal/fraud-report/index.html" rel="external"><?php _e('Report Trademark Abuse', 'onemozilla'); ?></a></li>
```
Comment (craigcook) [inline themes/OneMozilla/footer.php]: This link can be changed to 'https://www.mozilla.org/about/legal/fraud-report/'

**Comment by craigcook** [comment]
Good catch. Those pages aren't currently localized so you'll be directed to the English versions anyway, but we should still make the links locale-neutral for the sake of future-proofing. 

We can also update these links to their current URLs instead of the really old ".html" ones (which only still work because they're redirected on mozilla.org)

**Comment by pollti** [comment]
> Those pages aren't currently localized

There are localizations like https://www.mozilla.org/de/privacy/.

**Comment by craigcook** [comment]
I stand corrected, some of the policy pages are localized after all.

**Comment by craigcook** [comment]
Merged in 0c17a2f7f1233a9e787c23a0613e4af7c7622adc

**Comment by pollti** [comment]
Thank you very much!

**Pull request closed without merging**